README.md

```diff
@@ -41,29 +41,26 @@ Before you begin, make sure you have the following installed on your system:
     git clone https://github.com/citrineos/citrineos-core
     ```
 
-1. Navigate to the CitrineOS Server directory:
-
-    ```shell
-    cd citrineos-core/Server
-    ```
-
-1. Install project dependencies:
+1. Install project dependencies from root dir:
 
    ```shell
-   ./unix-init-install-all.sh
+   npm run install-all
    ```
 
-1. Start the server and its supporting infrastructure with:
+1. The docker container should be initialized by running `docker-compose -f ./Server/docker-compose.yml up -d` or 
+by using the IntelliJ `Server` Run Configuration which was created for this purpose.
```

Comment:
Sorry due to my very limited knowledge on typescript project, I am not exactly sure about what need to be done when reading this sentence. Would you mind explaining more for this part?

Reply:
@lydiazcheng take a look here https://www.jetbrains.com/help/idea/run-debug-configuration.html#templates it is particular to Intellij but it creates the runnable jobs in the IDE and the jobs essentially run various terminal commands to run docker or npm or attach debugger, etc.